stroyprombeton/tests/tests_views.py

```diff
@@ -619,35 +620,29 @@ def get_search_url(self, term=''):
             get_params=get_params.urlencode()
         )
 
-    def test_result_page_contains_query(self):
-        """Search results page should contain it's search query."""
-        url = self.get_search_url(term=self.WRONG_TERM)
-        response = self.client.get(url)
-        self.assertNotContains(response, self.WRONG_TERM)
+    def search(self, term=''):
+        return self.client.get(self.get_search_url(term), follow=True)
+
+    # @todo #626:30m Bring back a search query to search results page.
+    @unittest.skip
+    def test_search_query_on_result_page(self):
+        """The search query appears in search input of search results page."""
+        self.assertContains(
+            self.search(term=self.WRONG_TERM),
```

Comment:
let's add assert with `self.TERM` too?

Reply:
What do you mean? We use here `WRONG_TERM` because it won't find any results, that allow us to assert content of input. Probably it would be better to parse a response to find input content explicitly. It will clarify this test. I'll add this comment to the issue